validator/sawtooth_validator/journal/consensus/dev_mode/dev_mode_consensus.py

```diff
@@ -21,57 +24,117 @@
 from sawtooth_validator.journal.consensus.consensus\
     import ForkResolverInterface
 
+from sawtooth_validator.state.config_view import ConfigView
+
 
 class BlockPublisher(BlockPublisherInterface):
-    """ DevMode BlockPublisher implementation.
+    """Consensus objects provide the following services to the Journal:
+    1) Build candidate blocks ( this temporary until the block types are
+    combined into a single block type)
+    2) Check if it is time to claim the current candidate blocks.
+    3) Provide the data a signatures required for a block to be validated by
+    other consensus algorithms
```

Comment:
Please update this comment with the details of how it will work based on description in the jira task.